package/android/src/main/java/com/mrousavy/camera/CameraView+Events.kt

```diff
@@ -19,6 +19,20 @@ fun CameraView.invokeOnInitialized() {
   reactContext.getJSModule(RCTEventEmitter::class.java).receiveEvent(id, "cameraInitialized", null)
 }
 
+fun CameraView.invokeOnStarted() {
+  Log.i(CameraView.TAG, "invokeOnStarted()")
+
+  val reactContext = context as ReactContext
+  reactContext.getJSModule(RCTEventEmitter::class.java).receiveEvent(id, "cameraStarted", null)
```

Comment:
Hey @mrousavy Maybe we should probably add the native timestamp in this event? 
As otherwise, JS thread may delay a lot to consume these events, so if someone wants absolute precision he won't be able to do so in JS thread, making this feature less useful?

Reply:
Hey - why would you need the timestamp, what would that actually change?

Reply:
I can think of a feature where you want to synchronize video frames with another source, i.e bluetooth data from a device, in matter of ms delay.
To do so, one may want to have absolute control of when the video started/stopped recording frames, and match this with the device's stream input.
Without native timestamp on these events, we can only mark the timestamp on JS thread when this event is captured. Though, in case the JS thread is blocked and this event is processed later in the queue, we can end with completely false timestamp

Reply:
Tbh I don't agree here - this should definitely be done with a Frame Processor then, where you get a full stream of frames with their exactly accurate timestamps, and they are definitely in time as it is synchronous and blocking.